### ISSUE TITLE
BUG: sorting in `onyo get` is case insensitive

### ISSUE THREAD

**Issue by aqw** [psyinfra/onyo#651]
I'm choosing to file this as a bug rather than a feature request.

Sorting with `onyo get` currently is case insensitive:
```
❱ onyo get --keys make -s make
 ────── 
  make  
 ──────
...
  dell
  dell
  Dell
  dell
  dell
...
```
This interferes with getting an accurate list of all unique values for a particular field:
```
❱ onyo get --keys make -s make | uniq                
 ─────────────── 
  make           
 ─────────────── 
  apple          
  ctouch         
  dasgip         
  dell           
  Dell           
  dell           
  delta          
...
 ─────────────── 
```

**Comment by aqw** [comment]
I investigated this (and wrote a very nice upstream bug report, that I never submitted, as I discovered the solution first ;-).

The key is that we want to use `LOCALE` sorting rather than `INSENSITIVE`. This makes sense. Now the question is, do we:

- hardcode the locale to `en_US.UTF-8` (advantage is that onyo /always/ sorts the same on every machine)
- sniff for the OS's locale and use that (advantage: respect how the user has configured their machine)
